src/skin/legacy/tooltips.cpp

```diff
@@ -427,13 +427,41 @@ void Tooltips::addStandardTooltips() {
             << QString("%1: %2").arg(leftClick, tempoTapButton)
             << QString("%1: %2").arg(rightClick, bpmTapButton);
 
+    add("beats_set_change_marker")
+            << tr("Set Beat Marker")
+            << tr("Set a beat marker at the current play position.");
+
+    add("beats_remove_marker")
+            << tr("Remove Beat Marker")
+            << tr("Remove the beat marker at the current play position.");
+
+    add("beats_increase_bar_length")
+            << tr("Adjust the bar length")
+            << tr("Increment by one the bar length of the current marker.");
+
+    add("beats_decrease_bar_length")
+            << tr("Adjust the bar length")
+            << tr("Decrement by one the bar length of the current marker.");
+
     add("beats_adjust_slower")
             << tr("Adjust BPM Down")
-            << tr("When tapped, adjusts the average BPM down by a small amount.");
+            << tr("When tapped, decrease the BPM of the region around the "
```

Comment:
```suggestion
            << tr("When tapped, decrease the BPM between adjacent markers around the "
```
Here we have also the padding bar issue. Probably spit the right marker visyally and create a padding bar in between?